docs/knowledge/koor/koor-operator.md

```diff
@@ -0,0 +1,158 @@
+
+* Nodes: the cluster should contain at least **4** nodes.
+* Storage: the total storage available in the cluster should at least than **500GB**
+* CPU: across all nodes, the number of CPU cores should be at least **19 cores**
+* Memory: across all nodes, the available memory should be more than **44GB**
+
+To check if your cluster meets the minimum resources, you can check the status of the `KoorCluster` resource:
+
+```console
+$ kubectl describe -n koor-operator koorclusters
+...
+Status:
+  Meets Minimum Resources:  false
+  Total Resources:
+    Cpu:          6
+    Memory:       6052260Ki
+    Nodes Count:  3
+    Storage:      109293708Ki
+```
+
+## Version Notifications
+The Koor Operator allows you to be notified whenever a new version of KSD, Ceph or the Koor Operator are available. This is controlled in `values.yaml` using the following options:
+
+```yaml
+koorCluster:
+  spec:
+    upgradeOptions:
+      mode: notify # Change to diabled to turn off new version notifications
+      endpoint: https://versions.koor.tech # The endpoint for the version service
+      schedule: 0 0 * * * # The schedule to check for new versions, defaults to everyday at midnight
+```
+
+!!! tip
+    You can force an immediate version check by changing the `schedule` to `* * * * *` (check every minute) 
+    and changing it back to another more practical schedule when the check is complete.
+
+The current and latest versions will appear in the status of the `KoorCluster` resource:
+
+```console
+$ kubectl describe -n koor-operator koorclusters
+...
+Status:
+  Current Versions:
+    Ceph:           v17.2.6
+    Koor Operator:  0.3.5
+    Ksd:            v1.12.0
+    Kube:           v1.25.3
+  Latest Versions:
+    Ceph:
+      Image Hash:  9c067c50038de818e10ab7887929b6bd496d5dcfe55fa1343854a54e61a82fab
+      Image Uri:   quay.io/ceph/ceph:v17.2.6
+      Version:     17.2.6
+    Koor Operator:
+      Helm Chart:       koor-operator
+      Helm Repository:  https://koor-tech.github.io/koor-operator
+      Image Hash:       95b71899fcc90f8a34161d3f47a3ee3bde4fd11b6a4947c4482e3812ea89a019
+      Image Uri:        koorinc/koor-operator:v0.3.5
+      Version:          0.3.5
+    Ksd:
+      Helm Chart:           rook-ceph
+      Helm Repository:      https://charts.koor.tech/release
+      Image Hash:           d3d38ae93d268290bbf99545b90addaa6f51c3d1f383ef5f2ab6cb65f2cf243e
+      Image Uri:            koorinc/ceph:v1.12.0
+      Version:              1.12.0
+...
+```
+
+## Version Service
+The information about the newest available versions is tracked in the [version service](https://github.com/koor-tech/version-service). Clusters using the Koor Operator default to using the official Koor version service which is available on https://versions.koor.tech . 
+
+Alternatively, you can deploy the version service on your own cluster:
+
+```bash
+kubectl run version-service --image=koorinc/version-service --env="NO_TLS=true" --port 80 --expose
```

Comment:
Let's add a tag to the image so that people won't pull the moving `latest` tag by default, even if this might get outdated at some point, a version is easier to track than image "checksum" in the `kubectl describe pod` output.

```suggestion
kubectl run version-service --image=koorinc/version-service:IMAGE_TAG --env="NO_TLS=true" --port 80 --expose
```